src/tools/recipe2plan.ts

```diff
@@ -19,12 +19,12 @@ import {PlanGenerator} from './plan-generator.js';
  * @param scope kotlin package name
  * @return Generated Kotlin code.
  */
-export async function recipe2plan(path: string, scope: string): Promise<string> {
+export async function recipe2plan(path: string): Promise<string> {
   const manifest = await Runtime.parseFile(path);
```

Comment:
Will an error be thrown if there is no meta.namespace? 
If not, consider throwing an error in this generation.

Reply:
Added :)